### ISSUE TITLE
Constant docs have a mistake in an example

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6525]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#6525|
|Type|defect|
|Created|2013-05-24 00:03:26Z|

http://twistedmatrix.com/documents/12.2.0/core/howto/constants.html

Specifically, the `hasBody` example is backwards, it should be `not in`.

Attachments:

* [t6525-docs.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a23/a233355ade38932a82633cc3930e1d7f87aa67a9/aad82842ff64f5abae3c4e663aba1171b3b1fc2b.diff) (554 bytes) - added by Alex on 2013-05-24 00:13:28Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6525 6525
type__defect defect
reporter__Alex Alex
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1369353806000000 1369353806000000
changetime__1369355993000000 1369355993000000
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

(In [38458]) Apply t6525-docs.diff from Alex.

Refs: [#6525](https://github.com/twisted/twisted/issues/6525)

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)|@tomprince set owner to @tomprince|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_3"></a>|@tomprince commented|
|-|-|

Looks good. I'll merge.

This should really have a test, but we don't yet have the infrastructure for that. (See [#11340](https://github.com/twisted/twisted/issues/11340))

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_4"></a>|@tomprince set status to `closed`|
|-|-|

(In [38460]) Merge contants-doc-typo-6525: Fix typo in constants documentation.

Author: Alex
Reviewers: tom.prince
Fixes: [#6525](https://github.com/twisted/twisted/issues/6525)

The `hasBody` example was backwards, it should be `not in`.
